src/Homework4/Loor_back.py

```diff
@@ -0,0 +1,27 @@
+"""
+Оглянемся назад
+Даны два натуральных числа.
+Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+
+def evklid(a, b):
+    if a < b:
+        a, b = b, a
+    while a > 0:
+        if b <= 0:
+            break
+        a = a % b
+        if a <= 0:
+            break
+        b = b % a
+    if a > b:
```

Comment:
just a hack because you don't know where is solution. 
can be made more clear and simple

Reply:
corrected